neurons/miners/bitcoin/funds_flow/indexer.py

```diff
@@ -120,6 +120,54 @@ def move_forward(_bitcoin_node, _graph_creator, _graph_indexer, _graph_search, s
         else:
             logger.error(f"Failed to index block {block_height}.")
             time.sleep(30)
+            
```

Comment:
let merge it to 2.1 
do we need to update docs?

Reply:
changed it